digital_identity_client_test.go

```diff
@@ -0,0 +1,61 @@
+package yoti
+
+import (
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/getyoti/yoti-go-sdk/v3/dynamic"
+	"gotest.tools/v3/assert"
+)
+
+func TestDigitalIDClient(t *testing.T) {
+	key, readErr := os.ReadFile("./test/test-key.pem")
```

Comment:
I'm not sure why you create 2 different error objects (in the test below you create only one). It seems a bit unnecessary.

Reply:
Maybe but we have duplicated the tests in already done in client_test.go. so we used the same approach and test cases in client_test.go.  on the error we used the same approach.